stable/mercure/values.yaml

```diff
@@ -0,0 +1,56 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+allowAnonymous: "0"
+corsAllowedOrigins: ""
+debug: "0"
+demo: "0"
+jwtKey: ""
+logFormat: "FLUENTD"
+publishAllowedOrigins: ""
+publisherJwtKey: ""
+subscriberJwtKey: ""
+
+image:
+  repository: dunglas/mercure
+  tag: latest
```

Comment:
please do not use `latest` lets use a stable version

Reply:
done